tests/integration/targets/lookup_hashi_vault/playbooks/install_dependencies.yml

```diff
@@ -1,8 +1,8 @@
 - hosts: localhost
   tasks:
-    - name: Install openssl
-      import_role:
-        name: setup_openssl
+    - name: Install cryptography
+      pip:
+        name: cryptography
```

Comment:
Was this actually needed? I expected to do it in #2 but before adding this in, I just took out all the openssl stuff and nothing seemed to break?

Reply:
Not really, because it's already installed in the `default` container. I wouldn't rely on that though, that's why I explicitly added it.